components/sr-arena/component.html

```diff
@@ -34,42 +34,47 @@
 
         <sr-corner id="corner0" show="all"></sr-corner>
 
-        <div>
-            <sr-corner id="corner1" show="tla"></sr-corner>
-            <sr-corner id="corner2" show="tla"></sr-corner>
-            <sr-corner id="corner3" show="tla"></sr-corner>
-        </div>
+        <template if="{{ showOtherCorners }}">
+            <div>
+                <sr-corner id="corner1" show="tla"></sr-corner>
+                <sr-corner id="corner2" show="tla"></sr-corner>
+                <sr-corner id="corner3" show="tla"></sr-corner>
+            </div>
+        </template>
     </template>
 
     <script>
         Polymer({
             comp: null,
             arena: null,
             maincorner: 0,
+            showOtherCorners: true,
             arenaChanged: function() {
                 var corners = this.shadowRoot.querySelectorAll('sr-corner');
                 for (var i = 0; i < corners.length; i++) {
                     corners[i].arena = this.arena;
                 }
             },
             maincornerChanged: function() {
-                var otherCorners = [0, 1, 2, 3];
-                otherCorners.splice(this.maincorner, 1);
-
                 var corners = this.shadowRoot.querySelectorAll('sr-corner');
-                for (var i = 0; i < corners.length; i++) {
-                    if (i == 0) {
```

Comment:
Oh, I actually thought I had removed this before submitting the review when I realised...